my-app/src/app/modules/courses-page/core/pipes/orderBy/order-by.pipe.ts

```diff
@@ -2,7 +2,7 @@ import {Pipe, PipeTransform} from '@angular/core';
 
 import orderBy from "lodash/orderBy";
 
-import {Course} from "../../types";
+import {Course} from "../../";
```

Comment:
Looks like magick trick, when the type appears out of nowhere) Maybe putting the type to `shared` folder will make this path more readable.

Reply:
More over, this type uses in 'assets' folder too and it has to in shared folder, you're right. Fixed, added alias, thank you